server/api/core/email/email.js

```diff
@@ -27,27 +26,10 @@ export function send(options) {
  * @returns {Object} returns source
  */
 export function getTemplate(template) {
-  check(template, String);
```

Comment:
How to provide shop/language specific Email Template from the Templates collection, instead of the defaults from the file system with this update? How has the processing of those templates changed?